lib/config/wdtk-routes.rb

```diff
@@ -38,7 +41,22 @@
 
   get '/help/glossary' => 'help#glossary',
       as: :help_glossary
-  
+
   get '/help/environmental_information' => 'help#environmental_information',
       as: :help_environmental_information
+
+  get '/help/accessing_information' => 'help#accessing_information',
+      as: :help_accessing_information
+
+  get '/help/exemptions' => 'help#exemptions',
+      as: :help_exemptions
+
+  get '/help/authority_performance_tracking' => 'help#authority_performance_tracking',
+      as: :help_authority_performance_tracking
+
+  get '/help/search_engines' => 'help#search_engines',
+      as: :help_search_engines
+  
+   get '/help/freedom_of_information_scotland_act' => 'help#freedom_of_information_scotland_act',
```

Comment:
I'm not a fan of the long URL. Can we alias this, so a shorter version is also available?

Reply:
This was to keep it consistent with the new page at https://github.com/mysociety/whatdotheyknow-theme/pull/1728. If we change this, that will need modifying too.

Reply:
The people have spoken and picked short urls 🎉  I'll fix these across the two pages when I get a sec.